FAIL: test_cobyla.test_vector_constraints

With numpy 1.10-dev on 32-bit Linux:

    ======================================================================
    FAIL: test_cobyla.test_vector_constraints
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/rgommers/.local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
        self.test(*self.arg)
      File "/home/rgommers/Code/bldscipy/build/testenv/lib/python2.7/site-packages/scipy/optimize/tests/test_cobyla.py", line 100, in test_vector_constraints
        assert_allclose(sol, xsol, atol=1e-4)
      File "/home/rgommers/Code/numpy/numpy/testing/utils.py", line 1300, in assert_allclose
        verbose=verbose, header=header)
      File "/home/rgommers/Code/numpy/numpy/testing/utils.py", line 665, in assert_array_compare
        raise AssertionError(msg)
    AssertionError: 
    Not equal to tolerance rtol=1e-07, atol=0.0001

    (mismatch 50.0%)
     x: array([ 1.400112,  1.700056])
     y: array([ 1.4,  1.7])

Test precision only needs a small bump it looks like.

--- a/scipy/optimize/tests/test_cobyla.py
+++ b/scipy/optimize/tests/test_cobyla.py
@@ -96,21 +96,21 @@ def test_vector_constraints():
     fsol = 0.8
 
     # testing fmin_cobyla
-    sol = fmin_cobyla(fun, x0, cons_list, iprint=0)
+    sol = fmin_cobyla(fun, x0, cons_list, rhoend=1e-5, iprint=0)
     assert_allclose(sol, xsol, atol=1e-4)
 
-    sol = fmin_cobyla(fun, x0, fmin, iprint=0)
+    sol = fmin_cobyla(fun, x0, fmin, rhoend=1e-5, iprint=0)
     assert_allclose(fun(sol), 1, atol=1e-4)
 
     # testing minimize
     constraints = [{'type': 'ineq', 'fun': cons} for cons in cons_list]
-    sol = minimize(fun, x0, constraints=constraints)
+    sol = minimize(fun, x0, constraints=constraints, tol=1e-5)
     assert_allclose(sol.x, xsol, atol=1e-4)
     assert_(sol.success, sol.message)
     assert_allclose(sol.fun, fsol, atol=1e-4)
 
     constraints = {'type': 'ineq', 'fun': fmin}
-    sol = minimize(fun, x0, constraints=constraints)
+    sol = minimize(fun, x0, constraints=constraints, tol=1e-5)
     assert_allclose(sol.fun, 1, atol=1e-4)
 
 

